### PR TITLE
Episode of care measure icon size increase.

### PR DESCRIPTION
**Pull request by rbclark**
This increases the size of the episode of care measure icon size. @noranda is there a chance you could take a look at this and let me know if it matches what you were going for?

### CODE REVIEW

File: app/assets/stylesheets/_styles.scss [hunk 1]
```diff
@@ -26,6 +26,10 @@ $title-circle-radius: 21px;
     width: 1.28571429em;
     text-align: center;
   }
+  i.fa {
```
Comment (carpeliam) [inline app/assets/stylesheets/_styles.scss]: I'd recommend something like:

``` css
.measure-type {
  /* ... */
  .fa { /* ... */ }
  .glyphicon { /* ... */ }
}
```

if we need different rules for different types. That way you don't need to apply multiple rules for one type or another, and what you're trying to do will be more explicit.

**Comment by noranda** [comment]
The size looks great! I would however keep the width of the icons the same (for consistency). I realize this may break the alignment for episode-of-care measures, but we'll be condensing some of the columns shortly which will solve this.

**Comment by rbclark** [comment]
Thanks! Could you tell me what exactly is different with the with the width of the icons? From what I can tell they are still the same width in my browser. 

screen shot 2014-07-11 at 10 06 25 am

File: app/assets/stylesheets/_styles.scss (hunk 1 above)
Comment (rbclark) [inline app/assets/stylesheets/_styles.scss]: Sure, I can change that. Do we want to get rid of the generic i icon rule and just move all the code out into the two separate rules?

Comment (carpeliam) [inline app/assets/stylesheets/_styles.scss]: Yep, that works.

**Comment by carpeliam** [comment]
@rbclark It looks like you're trying to get the two widths to be equal, and specifying the width in em is misleading because it's based on font-size, which is different. It might be more clear here to specify the width in percentage, which is still responsive but not subject to font change, which in this case is variable between instances.

**Comment by rbclark** [comment]
Okay, I went in and tweaked it a bit, when I tried to set .fa and .glyphicon to a percentage width they the parent element .measure-type did not comply and ended up being a different size. I ended up setting the .measure-type to a set width and that seemed to work.

**Comment by carpeliam** [comment]
Cherry picked into #213.

**Pull request closed without merging**